README.rst

```diff
@@ -0,0 +1,101 @@
+Part of `edX code`__.
+
+__ http://code.edx.org/
+
+cookiecutter-django-ida
+=======================
+
+A cookiecutter_ template for edX Django projects.
+
+.. _cookiecutter: http://cookiecutter.readthedocs.org/en/latest/index.html
+
+
+This cookiecutter template is intended for new edX independently-deployable apps (IDAs). It includes the following packages:
+
+* Django 1.8.x
+* Django REST Framework
+* Django Waffle
+
+The necessary configuration is also in place to support:
+
+* i18n,
+* documentation (using Sphinx_),
+* authentication with edX Open ID Connect,
+* loading settings from YAML (for production),
+* Pylint/PEP8 validation,
+* and Travis CI.
+
+.. _Sphinx: http://sphinx-doc.org/
+
+Usage
+-----
+As with any new project, you will need to create a virtual environment. Once this is setup, install cookiecutter and
+edx-lint:
+
+.. code-block:: bash
+
+    $ pip install cookiecutter
+    $ pip install git+https://github.com/edx/edx-lint.git@v0.2.5#egg=edx_lint==0.2.5
+
+cookiecutter has the ability to pull templates directly from git, so there is no need to clone this repo. Simply
+provide the repo path as an argument:
+
+.. code-block:: bash
+
+    $ cd <workspace>
+    $ cookiecutter  https://github.com/edx/cookiecutter-django-ida.git
+
+You will be prompted for a few basic details (described below). These will be used to create the new project.
+
++---------------+-----------------------------------------------------------------------------------------------------------------------+
```

Comment:
Not sure if there's a technical reason to create tables this way, but for future reference, there's a much easier way.

    .. list-table::
       :widths: 25 50
       :header-rows: 1

     * - Name
       - Description   
     * - Conditional Module (Poll)
       - You can create a conditional module to control versions of content that
         groups of students see. 
     * - JavaScript Problem
       - Custom JavaScript display and grading problems allow you to create a
         custom problem or tool that uses JavaScript.